architecture/porting_guidelines.rst

```diff
@@ -58,9 +58,9 @@ variables (``CFG_``), which may be assigned values in two ways. ``CFG_FOO ?=
 bar`` should be used to provide a default value that may be modified at compile
 time. On the other hand, variables that must be set to some value and cannot be
 modified should be set by: ``$(call force,CFG_FOO,bar)``. - Compiler flags for
-the TEE core, the user mode libraries and the Trusted Applications, which may be
-added to macros used by the build system. Please see `Platform-specific
-configuration and flags`_ in the build system documentation.
+the TEE core, the user mode libraries and the Trusted Applications, which may
+be added to macros used by the build system. Please see
+:ref:`configuration_and_flags` and similar sections on that page.
```

Comment:
Should the line 436 be removed also?

Reply:
Yes, you're right, I missed that since I thought the link was to a section, good catch!